packages/db/bin/server.js

```diff
@@ -0,0 +1,26 @@
+const { ApolloServer } = require("apollo-server");
+
+const { TruffleDB } = require("@truffle/db");
+const Config = require("@truffle/config");
+
+const port = 4444;
+
+const config = Config.detect({
+  workingDirectory: process.argv[2] || process.cwd()
+});
+
+const db = new TruffleDB(config);
+
+const { schema, context } = db;
+
+const server = new ApolloServer({
+  tracing: true,
```

Comment:
What is `tracing`?

Reply:
Nevermind! https://github.com/apollographql/apollo-server/tree/HEAD/packages/apollo-tracing